### PR TITLE
core(fr): convert inspector-issues gatherer

### PR DESCRIPTION
**Pull request by adamraine**
Ref #11313

### CODE REVIEW

File: lighthouse-core/test/fraggle-rock/api-test-pptr.js [hunk 1]
```diff
@@ -51,7 +51,9 @@ describe('Fraggle Rock API', () => {
     server = new StaticServer();
     await server.listen(0, '127.0.0.1');
     serverBaseUrl = `http://localhost:${server.getPort()}`;
+    if (!process.env.CHROME_PATH) throw Error('CHROME_PATH is not set');
     browser = await puppeteer.launch({
+      executablePath: process.env.CHROME_PATH,
```
Comment (adamraine) [inline lighthouse-core/test/fraggle-rock/api-test-pptr.js:56]: `Audits` CDP domain is not available in the Chromium which ships with our current puppeteer.

Comment (connorjclark) [reply lighthouse-core/test/fraggle-rock/api-test-pptr.js:56]: I think we can land #12284 soon.

File: lighthouse-core/gather/gatherers/inspector-issues.js [hunk 2]
```diff
@@ -28,25 +36,28 @@ class InspectorIssues extends Gatherer {
...
 
   /**
-   * @param {LH.Gatherer.PassContext} passContext
+   * @param {LH.Gatherer.FRTransitionalContext} context
    */
-  async beforePass(passContext) {
-    const driver = passContext.driver;
-    driver.on('Audits.issueAdded', this._onIssueAdded);
-    await driver.sendCommand('Audits.enable');
+  async startSensitiveInstrumentation(context) {
```
Comment (patrickhulce) [inline lighthouse-core/gather/gatherers/inspector-issues.js:41]: I'm thinking this one isn't sensitive. Any compelling reason to keep in sensitive?

File: lighthouse-core/test/fraggle-rock/api-test-pptr.js (hunk 1 above)
Comment (patrickhulce) [reply lighthouse-core/test/fraggle-rock/api-test-pptr.js:56]: +1 to waiting for #12284 and/or just marking as an OK failure. #12510 has the same problem

**Review by patrickhulce — COMMENTED** [review]
nice! you're flying through these 😃

File: lighthouse-core/gather/gatherers/inspector-issues.js (hunk 2 above)
Comment (adamraine) [reply lighthouse-core/gather/gatherers/inspector-issues.js:41]: We've had CSP issues logged for things Lighthouse injects into the page https://github.com/GoogleChrome/lighthouse/issues/11862

Comment (patrickhulce) [reply lighthouse-core/gather/gatherers/inspector-issues.js:41]: If that's it then `startInstrumentation` should be fine. Both instrumentation and sensitiveInstrumentation end before `getArtifact`. sensitiveInstrumentation is only a concern if the process of *ending* other instrumentation could show up in results.

File: lighthouse-core/test/fraggle-rock/api-test-pptr.js (hunk 1 above)
Comment (adamraine) [reply lighthouse-core/test/fraggle-rock/api-test-pptr.js]: Should be good now.

**Review by patrickhulce — APPROVED** [review]

**Pull request merged**